crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -1944,20 +1952,9 @@ where
                 }
 
                 self.semantic.handled_exceptions.push(handled_exceptions);
-
-                if self.enabled(Rule::JumpStatementInFinally) {
-                    flake8_bugbear::rules::jump_statement_in_finally(self, finalbody);
-                }
-                if self.enabled(Rule::ContinueInFinally) {
-                    if self.settings.target_version <= PythonVersion::Py38 {
-                        pylint::rules::continue_in_finally(self, finalbody);
-                    }
-                }
```

Comment:
Also moving these up to the "pre-visit" phase with all the other rules -- no reason for them to be here, in the "recurse" phase.